client/src/public/ts/enemies.ts

```
@@ -224,11 +224,11 @@ function getBestFontSize(decrement: number, text: string, width: number) {
   let size = ENEMY_FONT_SIZE * variables.settings.enemySizeCoefficient;
   let style = _.clone(ENEMY_TEXT_STYLE);
   while (size > 12) {
-    let textMetrics = PIXI.TextMetrics.measureText(text, style);
+    let textMetrics = PIXI.CanvasTextMetrics.measureText(text, style);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'textMetrics' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)